src/biotite/database/alphafold/download.py

```diff
@@ -0,0 +1,121 @@
+# This source code is part of the Biotite package and is distributed
+# under the 3-Clause BSD License. Please see 'LICENSE.rst' for further
+# information.
+
+__name__ = "biotite.database.alphafold"
+__author__ = "Alex Carlin"
+__all__ = ["fetch"]
+
+from os.path import isdir, isfile, join, getsize
+import os
+import io
+import requests
+from .check import assert_valid_response
+
+
+_fetch_url = "https://alphafold.com/api/prediction"
+
+
+def fetch(ids, target_path=None, format="pdb", overwrite=False, verbose=False):
```

Comment:
This purely optional, but I think it would be nice to directly support the Alphafold identifiers as well. Furthermore, accepting the Alphafold identifiers given from the PDB computational models would even enable synergy between both, the `rcsb` and `alphafold` interfaces.

While using official Alphafold identifers should be straight forward (only the `v4` would need to be updated when a new version appears), using the identifier from the PDB would require some string editing. Taken from the [PDB documentation](https://www.rcsb.org/docs/general-help/computed-structure-models-and-rcsborg):

> Each CSM is assigned a specific ID in its source database and a prefix indicates the source of the model (“AF” for [AlphaFold DB](https://alphafold.ebi.ac.uk/), "MA" for [ModelArchive](https://www.modelarchive.org/)). AlphaFold DB identifiers are then followed by the UniProt accession number for the protein and by the fragment number (usually “F1”). However, in order to enable compatibility of the IDs with many of our services, including all of our APIs and visualization tools, we identify CSMs on RCSB.org using a modified version of the ID. This ID is used on the structure summary page, in searching for structures, in the search results page, and in various tools for 3D structure visualization and analysis. For example, for the AlphaFold structure AF-B3EWR1-F1, the RCSB.org assigned CSM ID is AF_AFB3EWR1F1 and is used in the query results page as shown in Figure 4.

Which type of identifier is present can be detected from whether it starts with `AF_` (PDB), `AF-` (Alphafold DB) or something else (Uniprot).

I would let you decide if we want this feature directly from the beginning